code/controllers/subsystem/ticker.dm

```diff
@@ -472,9 +472,9 @@ SUBSYSTEM_DEF(ticker)
 		addtimer(CALLBACK(src, PROC_REF(release_characters), livings), 30, TIMER_CLIENT_TIME)
 
 /datum/controller/subsystem/ticker/proc/release_characters(list/livings)
-	for(var/I in livings)
-		var/mob/living/L = I
-		L.notransform = FALSE
+	for(var/iterable in livings)
+		var/mob/living/living_mob = iterable
```

Comment:
May as well make this an as anything loop if you're already giving it the definitive edition